### PR TITLE
OCP 4.14 RN's: add HCP support for P/Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.14

Requires CM

Issue: 
https://issues.redhat.com/browse/OSDOCS-6732

Link to docs preview: 
https://68637--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-ibm-power

Related PR: https://github.com/openshift/openshift-docs/pull/68627
- Associated Jira: https://issues.redhat.com/browse/MULTIARCH-3703

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -2707,6 +2707,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |General Availability
 
+|Hosted control planes for {product-title} on ARM
```
Comment (lahinson) [inline release_notes/ocp-4-14-release-notes.adoc:2710]:
```suggestion
|Hosted control planes for an ARM64 {product-title} cluster on AWS
```

**Review by lahinson — COMMENTED** [review]
@alishaIBM I added one revision based on feedback from the PM and developer who worked on the ARM docs. Thank you!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (alishaIBM) [reply release_notes/ocp-4-14-release-notes.adoc]: Thank you @lahinson 
I have updated the PR as per above. Thanks.

**Review by lahinson — APPROVED** [review]
LGTM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1294,10 +1294,14 @@ Hosted control planes for {product-title} is now Generally Available on the bare
 ==== Creating ARM NodePool objects on AWS hosted clusters (Technology Preview)
 In this release, you can schedule application workloads on 64-bit ARM and AMD64 from the same hosted control plane. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#hosted-cluster-arm-node-pools[Creating ARM NodePool objects on AWS hosted clusters].
 
-[id="ocp-4-14-hcp-ibm-power-z"]
+[id="ocp-4-14-hcp-ibm-z"]
 ==== Hosted control planes on {ibm-z-title} (Technology Preview)
 In this release, hosted control planes is available as a Technology Preview feature on {ibm-z-title}. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-ibmz[Configuring the hosting cluster on 64-bit x84 bare metal for IBM Z compute nodes (Technology Preview)].
 
+[id="ocp-4-14-hcp-ibm-power"]
```
Comment (lahinson) [inline release_notes/ocp-4-14-release-notes.adoc:1301]: @alishaIBM @SNiemann15 The content in lines 1297 - 1299 is currently under review in PR #68676. You can remove those lines from this PR. Thank you!

**Comment by manojnkumar** [comment]
LGTM

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (alishaIBM) [reply release_notes/ocp-4-14-release-notes.adoc]: @lahinson okay, I have updated the PR. cc: @SNiemann15

Comment (lahinson) [reply release_notes/ocp-4-14-release-notes.adoc]: Thank you, @alishaIBM!

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by alishaIBM** [comment]
PTAL: @sferich888, @davemulford, @xltian, @kalexand-rh, @sjstout

**Comment by davemulford** [comment]
LGTM

**Comment by SNiemann15** [comment]
> /label merge-review-needed

Please don't merge yet. This PR requires CM and the [Notice of Change] mail hasn't gone out yet.

**Comment by abhatt-rh** [comment]
> LGTM! Can you please update the preview link to something like https://68637--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-ibm-z ?
> 
> I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

LGTM!

**Pull request merged**